bib/go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/aws/aws-sdk-go v1.52.1
 	github.com/cheggaaa/pb/v3 v3.1.5
 	github.com/google/uuid v1.6.0
-	github.com/osbuild/images v0.58.0
+	github.com/osbuild/images v0.58.1-0.20240430053854-ddf3ea9f0280
```

Comment:
Is this an untagged commit ID?  I did a v0.59 release if you'd rather pull that in.

Reply:
Sure thing! Fixed now :)